samples/AppSecretsConfig/Pages/Shared/_ValidationScriptsPartial.cshtml

```diff
@@ -0,0 +1,2 @@
+<script src="~/lib/jquery-validation/dist/jquery.validate.min.js"></script>
```

Comment:
Showing your age with the jQuery references, eh?  😛 

_(I don't have a better suggestion; I'm way out of touch with the modern client-side frameworks)_

Reply:
It was from the template (`dotnet new webapp`). 😊 I modified it only enough to remove the privacy link, header, and footer since they were unnecessary and I didn't want to have some false privacy policy that didn't align with Microsoft's, which didn't seem necessary here since we're not even deploying or running the sample itself.

Reply:
Besides, I think jQuery is still used underpinning many libraries that I've seen. 😉